README.md

```diff
@@ -17,27 +17,62 @@ Integration branch status: [![Integration Branch Build Status](https://travis-ci
 
 This repository acts as the source code location for [the Unicorn Utterances blog](https://unicorn-utterances.com)
 
-## Important Files
+## Sponsors
+
+[![The Polyglot Developer](./static/sponsors/the-polyglot-developer.svg)](https://www.thepolyglotdeveloper.com/)
+
+## Statement of Ethics
+
+We never want to end up in a place where our educational content or the community or
+experience around that content is compromised. Not by money and not by any potential
+of harmful members within said community. As a result,
+[we've implemented the Contributor Covenant as our code of conduct](CODE_OF_CONDUCT.md).
+
+We also pledge to keep transparent communication in regards to finances
+that flow through the project. While not every sponsorship contains a
+financial contribution, if one does it will be disclosed both what those
+finances are going towards as well as what will be done in exchange for
+said contribution will be clearly and transparently laid out.
```

Comment:
```suggestion
We also pledge to maintain transparency in regards to any finances that flow
through the project. Not every sponsorship contains a financial contribution,
but for ones that do, it will be clearly disclosed what those finances are
going towards as well as what will be done in exchange for that contribution.
```